index.html

```diff
@@ -339,73 +340,62 @@ <h3>Methods</h3>
       </dl>
     </section>
   </section>
-  <section id="rtcrtpreceiver-playoutdelay">
+  <section id="rtcrtpreceiver-jitterbufferdepth">
     <h3>
-      Controlling the received playout delay
+      Adjusting the receiver's jitter buffer
     </h3>
     <p>
       The {{RTCRtpReceiver}}
       interface is defined in [[WEBRTC]]. This document extends that interface
-      by adding an additional internal slot and attribute for the playoutDelay.
+      by adding an additional attribute to adjust the receiver's jitter buffer.
     </p>
-    <section id="rtcrtpreceiver-playoutdelay-modifications">
+    <section id="rtcrtpreceiver-jitterbufferdepth-modifications">
       <h2>Modification to existing procedures</h2>
       <p>
-        Let {{RTCRtpReceiver}} objects have a <dfn data-dfn-for=RTCRtpReceiver>[[\PlayoutDelay]]</dfn> internal slot initially
+        Let {{RTCRtpReceiver}} objects have a <dfn data-dfn-for=RTCRtpReceiver>[[\JitterBufferDepth]]</dfn> internal slot initially
         initialized to <code>null</code>.
       </p>
     </section>
-    <section id="rtcrtpreceiver-playoutdelay-rtcrtpreceiver-interface">
+    <section id="rtcrtpreceiver-jitterbufferdepth-rtcrtpreceiver-interface">
       <h2>{{RTCRtpReceiver}} interface extensions</h2>
       <pre class="idl">partial interface RTCRtpReceiver {
-  attribute double? playoutDelay;
+  attribute double? jitterBufferDepth;
```

Comment:
Should we use DOMHighResTimestamp instead? It's also a double but used for milliseconds and this is ms now.

> The DOMHighResTimeStamp type is a double and is used to store a time value in milliseconds

Reply:
We could, but it's not a timestamp, so that seems confusing. It's only mentioned under [§ 8.4. Use the appropriate type to represent times and dates](https://w3ctag.github.io/design-principles/#times-and-dates).